renderer.js

```diff
@@ -2075,19 +2079,55 @@ class SDFGRenderer {
         });
         // Mark hovered and highlighted elements.
         this.for_all_elements(this.mousepos.x, this.mousepos.y, 0, 0, (type, e, obj, intersected) => {
+            // Highligh all edges of the memlet tree
             if (intersected && obj instanceof Edge && obj.parent_id != null) {
-                let tree = memlet_tree(e.graph, obj);
+                let tree = memlet_tree_complete(this.graph, obj, this.recompute_memlet_trees);
+                this.recompute_memlet_trees = false;
                 tree.forEach(te => {
                     if (te != obj) {
                         te.highlighted = true;
                     }
                 });
             }
+            
+            // Highlight all access nodes with the same name in the same nested sdfg
+            if (intersected && obj instanceof AccessNode) {
+                this.for_all_elements(0, 0, 0, 0, (type, e, sobj, intersected) => {
```

Comment:
It would be faster to just loop over `this.sdfg_list[obj.sdfg.sdfg_list_id]`

Reply:
Yes, I changed that now